pyproject.toml

```diff
@@ -43,9 +41,6 @@ furo = { version = "^2023.9.10", optional = true }
 myst-parser = { version = "^2.0.0", optional = true }
 sphinx = { version = "^7.2.5", optional = true }
 
-[tool.poetry.group.crawler_s3.dependencies]
```

Comment:
Leave this group in; I think it will let me install only crawler_s3 dependencies for those Dockerfiles.